test/functional/Valhalla/src_lw5/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -1797,4 +2536,305 @@ static public void createClassWithVolatileValueTypeFields() throws Throwable {
 		ClassWithValueTypeVolatile c = ClassWithValueTypeVolatile.createObjectWithDefaults();
 		c.checkFieldsWithDefaults();
 	}
+
+	@Test(priority=1, expectedExceptions=ClassFormatError.class)
+	static public void testValueTypeHasSynchMethods() throws Throwable {
+		String fields[] = {"longField:J"};
+		Class valueClass = ValueTypeGenerator.generateIllegalValueClassWithSynchMethods("TestValueTypeHasSynchMethods", fields);
+	}
+
+	//*******************************************************************************
+	// default value tests
+	//*******************************************************************************
+
+	/*
+	 * Create a value type and read the fields before
+	 * they are set. The test should Verify that the
+	 * flattenable fields are set to the default values.
+	 * NULL should never be observed for Qtypes.
+	 */
+	@Test(priority=4)
+	static public void testDefaultValues() throws Throwable {
+		AssortedValueWithLongAlignment assortedValueWithLongAlignment = new AssortedValueWithLongAlignment();
+		assertNotNull(assortedValueWithLongAlignment.point);
+		assertNotNull(assortedValueWithLongAlignment.line);
+		assertNotNull(assortedValueWithLongAlignment.o);
+		assertNotNull(assortedValueWithLongAlignment.l);
+		assertNotNull(assortedValueWithLongAlignment.d);
+		assertNotNull(assortedValueWithLongAlignment.i);
+		assertNotNull(assortedValueWithLongAlignment.tri);
+
+		/* Test with assorted ref object with long alignment */
+		AssortedRefWithLongAlignment assortedRefWithLongAlignment = new AssortedRefWithLongAlignment();
+		assertNotNull(assortedRefWithLongAlignment.point);
+		assertNotNull(assortedRefWithLongAlignment.line);
+		assertNotNull(assortedRefWithLongAlignment.o);
+		assertNotNull(assortedRefWithLongAlignment.l);
+		assertNotNull(assortedRefWithLongAlignment.d);
+		assertNotNull(assortedRefWithLongAlignment.i);
+		assertNotNull(assortedRefWithLongAlignment.tri);
+
+		/* Test with flattened line 2D */
+		FlattenedLine2D flattenedLine2D = new FlattenedLine2D();
+		assertNotNull(flattenedLine2D.st);
+		assertNotNull(flattenedLine2D.en);
+
+		/* Test with triangle 2D */
+		Triangle2D triangle2D = new Triangle2D();
+		assertNotNull(triangle2D.v1);
+		assertNotNull(triangle2D.v2);
+		assertNotNull(triangle2D.v3);
+	}
+
+	/*
+	 * Create Array Objects with Point Class without initialization
+	 * The array should be set to a Default Value.
+	 */
+	@Test(priority=4, invocationCount=2)
+	static public void testDefaultValueInPointArray() throws Throwable {
+		Point2D[] pointArray = new Point2D[genericArraySize];
```

Comment:
From the description of the PR `All ValueTypeTests with lw5 are passing except for testDefaultValueIn*Array tests`, we seem to fail this test now. But I think the array used here should be `Point2D![]`.

Reply:
I was thinking it shouldn't matter because the implicitcreation default flag determines default values, not flattening.

Reply:
Maybe we should check the behaviour of this on RI, i.e. the default value of element in Point2D[].

Reply:
Just tried with my lw5 builds and it looks like this behavior is not implemented yet.

Reply:
So RI is having default null elements for array created by `new Point2D[size]` and non-null elements for array created by `Point2D![size]` ?

Reply:
I will add a comment here to double check the behavior for this once ri is more up to date.

Reply:
> So RI is having default null elements for array created by new Point2D[size] and non-null elements for array created by Point2D![size] ?

At the moment it is null for both cases